src/Build/Logging/BaseConsoleLogger.cs

```diff
@@ -29,6 +30,13 @@ internal abstract class BaseConsoleLogger : INodeLogger
         /// </summary>
         private static bool _supportReadingBackgroundColor = true;
 
+        ///<summary>
+        /// TargetFramework_mapping is the map between a (nodeID and project_context_id) to a target framework
+        /// TargetFramework_errorwarning is the map between a target framwork and the number of errors and warnings
+        /// These are used to give more direction as to where an error is happening when multi targetted
+        ///</summary>
+        protected Dictionary<(int nodeId, int contextId), Dictionary<string, string>> TargetFramework_mapping = new Dictionary<(int nodeId, int contextId), Dictionary<string, string>> ();
```

Comment:
Originally it was necessary because each thread has multiple properties, since it's not just the TargetFrameworks we are printing and each thread could be using different properties. I can just skip the second mapping and build the string originally to simplify it to one map